main/EDMainMod.F90

```diff
@@ -927,26 +917,34 @@ subroutine ed_update_site( currentSite, bc_in, bc_out, is_restarting )
     ! Set boundary condition to HLM for carbon loss to atm from fires and grazing
     ! [kgC/ha/day]*[ha/m2]*[day/s] = [kg/m2/s] 
     
-    bc_out%fire_closs_to_atm_si = site_cmass%burn_flux_to_atm * ha_per_m2 * days_per_sec
-    bc_out%grazing_closs_to_atm_si = site_cmass%herbivory_flux_out * ha_per_m2 * days_per_sec
-
-    
+    bc_out%fire_closs_to_atm_si = site_cmass%burn_flux_to_atm * area_inv * days_per_sec
+    bc_out%grazing_closs_to_atm_si = site_cmass%herbivory_flux_out * area_inv * days_per_sec
+    bc_out%gpp_site = site_cmass%gpp_acc * area_inv * days_per_sec
+    bc_out%ar_site  = site_cmass%aresp_acc * area_inv * days_per_sec
 
   end subroutine ed_update_site
 
   !-------------------------------------------------------------------------------!
 
-  subroutine TotalBalanceCheck (currentSite, call_index )
+  subroutine TotalBalanceCheck (currentSite, call_index, is_restarting )
```

Comment:
Can this be an optional argument? With a local variable that is .false. by default and takes the argument value when the argument is present? Then you don't have to alter every call of this routine